hip-service/Middleware/MiddlewareExtensions.cs

```diff
@@ -0,0 +1,13 @@
+using Microsoft.AspNetCore.Builder;
+
+namespace hip_service.Middleware
+{
+    public static class MiddlewareExtensions
+    {
+        public static IApplicationBuilder UseSecurityHeadersMiddleware(this IApplicationBuilder app)
```

Comment:
Rename this middleware to `UseConsentManagerIdentifierMiddleware` or on the similar lines?

Reply:
done